samples/ck-rx65n/basic-sample/include/app_config.h

```diff
@@ -32,6 +32,6 @@
 #define ENABLE_RX_TSIP_AUTH_DRIVER_SAMPLE
 #endif // IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
 
-#define SAMPLE_SNTP_SERVER_NAME "time.google.com"    /* SNTP Server.  */
+#define SAMPLE_SNTP_SERVER_NAME "time.apple.com"    /* SNTP Server.  */
```

Comment:
Intentional?

Reply:
Yes, Michael asked for this in this ticket - https://iotconnectsdk.atlassian.net/browse/IOTSDK01-389?atlOrigin=eyJpIjoiMTVhNWU1MTZiYTNmNGY3ZDkwNTI0ZjMyMTNiMGFjM2YiLCJwIjoiaiJ9